Mathlib/FieldTheory/KrullTopology.lean

```diff
@@ -210,77 +163,64 @@ theorem IntermediateField.fixingSubgroup_isClosed {K L : Type*} [Field K] [Field
     IsClosed (E.fixingSubgroup : Set (L ≃ₐ[K] L)) :=
   OpenSubgroup.isClosed ⟨E.fixingSubgroup, E.fixingSubgroup_isOpen⟩
 
+/-- For a field extension `L/K`, the Krull topology on `L ≃ₐ[K] L` makes it a
+  nonarchimedean group. -/
+instance (K L : Type*) [Field K] [Field L] [Algebra K L] : NonarchimedeanGroup (L ≃ₐ[K] L) :=
+  fixingSubgroup_isGroupBasis K L |>.nonarchimedean_of_subgroups
+
+/-- Given a field extension `L/K`, `finiteExts K L` is the set of
+intermediate field extensions `L/E/K` such that `E/K` is finite -/
+def finiteExts (K : Type*) [Field K] (L : Type*) [Field L] [Algebra K L] :
+    Set (IntermediateField K L) :=
+  {E | FiniteDimensional K E}
+
+/-- Given a field extension `L/K`, `fixedByFinite K L` is the set of
+subsets `Gal(L/E)` of `L ≃ₐ[K] L`, where `E/K` is finite -/
+def fixedByFinite (K L : Type*) [Field K] [Field L] [Algebra K L] : Set (Subgroup (L ≃ₐ[K] L)) :=
+  IntermediateField.fixingSubgroup '' finiteExts K L
+
+/-- If `L/K` is a field extension, then we have `Gal(L/K) ∈ fixedByFinite K L` -/
+theorem top_fixedByFinite {K L : Type*} [Field K] [Field L] [Algebra K L] :
+    ⊤ ∈ fixedByFinite K L :=
+  ⟨⊥, IntermediateField.finiteDimensional_bot K L, IntermediateField.fixingSubgroup_bot⟩
```

Comment:
I think we can safely remove these, but I want some external opinion first.

Reply:
Can you elaborate? 
Does it hurt to just  have them be deprecated for the moment?

Reply:
Sure, deprecating doesn't cost much.

The reasoning is the following : as I interpret it, the only reason we wanted to consider `fixedByFinite` as a set of subgroups is because [GroupFilterBasis](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/FilterBasis.html#GroupFilterBasis) asks for a set of set, so there was no way to avoid turning the (naturally indexed) family into a set. If that was indeed the only motivation behind these definitions, then they become irrelevant since it's now easier to work with the indexed family. But I usually don't interact at all with that part of the library, so I'm not the right person to decide what is useful or not.